SYCL/Plugin/sycl-ls.cpp

```diff
@@ -12,3 +12,7 @@
 // The test crashed on CUDA CI machines with the latest OpenCL GPU RT
 // (21.19.19792).
 // UNSUPPORTED: cuda
+
```

Comment:
please remove this change and add // UNSUPPORTED: esimd_emulator at the top

Reply:
Will do.

Reply:
Done.